project-2/HPFpreemptive.cpp

```diff
@@ -207,10 +206,6 @@ overStat HPFpre_emptive (List * input_list)
     
     std::cout << "*************** OVERALL STATISTICS FOR  ALL QUEUE *************";
     printAlgoStats(totalResponseTime, totalTurnarounTime, totalWaitTime, numProcessedJobs);
-
```

Comment:
tmp was assigned instead of allocating with 'new' 
so I will just let the system clean up the memory.